core/src/main/java/org/apache/calcite/util/Bug.java

```diff
@@ -174,6 +182,18 @@
    * JSON data type support</a> is fixed. */
   public static final boolean CALCITE_2869_FIXED = false;
 
+  /** Whether
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-3243">[CALCITE-3243]
+   * Incomplete validation of operands in JSON functions</a> */
```

Comment:
minor: missing "is fixed" at the end of this comment (CALCITE-3243), same for the comment below (CALCITE-3245).

Reply:
Thanks!